### ISSUE TITLE
Was a Custom Soundtrack ever considered for this game?

### ISSUE THREAD

**Issue by SecularSteve** [juzzlin/DustRacing2D#123]
Just asking in case you're open to having custom MIDIs made in LMMS or something...

**Comment by juzzlin** [comment]
Yes, I planned composing a menu music + a couple of actual game tunes, but never had time to do that. Ogg Vorbis would be the best format here. However, the biggest problem would be possible copyright issues as the music being used should be public domain.

**Comment by SecularSteve** [comment]
That was the point anyway.
I asked in case you're looking for someone to make it.

**Comment by juzzlin** [comment]
Well, not actively looking, but could accept something appropriate :)

**Comment by SecularSteve** [comment]
Here it is.

[DustRacingMenu.zip](https://github.com/juzzlin/DustRacing2D/files/8484494/DustRacingMenu.zip)

Loop Sample:
https://www.sndup.net/fr7x/d

I worked on it purely for fun, because I was bored, and because I thought that this game needed some life with at least one track.

It's made using nothing but Vanilla LMMS assets, and it's as Public Domain as you can get.

All I ask is that you credit me as "Menu Theme by SecularSteve" if you could (only if it's good enough first).

**Comment by SecularSteve** [comment]
Is everything alright?

**Comment by juzzlin** [comment]
Sorry, I've been too busy with my life lately :( 

Would it be possible to render this with a proper synth? I'm not a real musician, but I'd prefer audio quality comparable to this old tune of mine: https://www.youtube.com/watch?v=53diIj0kFVg

**Comment by SecularSteve** [comment]
Hello again :)
Are you still open to this proposal?
